lms/templates/logout.html

```diff
@@ -5,14 +5,16 @@
 {% block title %}{% trans "Signed Out" as tmsg %}{{ tmsg | force_escape }} | {{ block.super }}{% endblock %}
 
 {% block body %}
-    <h1>{% trans "You have signed out." as tmsg %}{{ tmsg | force_escape }}</h1>
+    {% if enterprise_target is False %}
```

Comment:
@ziafazal Can you explain this change?  Showing a blank screen is not the latest decision on the ticket: https://openedx.atlassian.net/browse/ENT-1688

This current implementation may be even more confusing to the enterprise learner.  It seems to me that there's an underlying issue with the enterprise SSO flow.  Why would the learner be logged out during SSO login?